x-pack/plugins/ml/server/models/job_audit_messages/job_audit_messages.js

```diff
@@ -123,7 +127,10 @@ export function jobAuditMessagesProvider({ asInternalUser }, mlClient) {
 
     let messages = [];
     if (body.hits.total.value > 0) {
-      messages = body.hits.hits.map((hit) => hit._source);
+      messages = body.hits.hits.map((hit) => ({
+        clearable: hit._index === ML_NOTIFICATION_INDEX_02,
```

Comment:
This is fragile, as it will stop working when there's a reason to have `.ml-notifications-000003`.  It would be more future-proof to say an index is clearable if it ends with a 6 digit number and that number is >= 2.

Reply:
Created an issue for 7.15 to address this https://github.com/elastic/kibana/issues/103742